src/mpid/ch4/generic/mpidig_globals.c

```diff
@@ -47,7 +47,7 @@ int MPIDIG_comm_abort(MPIR_Comm * comm, int exit_code)
             continue;
 
         mpi_errno = MPI_SUCCESS;
-        sreq = MPIDI_CH4I_am_request_create(MPIR_REQUEST_KIND__SEND, 2);
+        sreq = MPIDI_am_request_create(MPIR_REQUEST_KIND__SEND, 2);
```

Comment:
I kind of think this is `MPIDIG` or even `MPIDIGI` functionality, since it's internal to the generic component. Adding another "I" on the end seems silly, though. What do you think?

Reply:
I agree it is just for `MPIDIG`. I left it in the `MPIDI` because it is calling the netmod and shmmod level function. We can discuss this with everyone over the call, I guess.

Reply:
I think I agree with Ken here. It's legal for `MPIDIG` to call netmod/shmod functionality. In fact, most of the time it does.